query/functions/signature.feature

```diff
@@ -0,0 +1,219 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#noinspection CucumberUndefinedStep
+Feature: Effects a function signature have on the caller
```

Comment:
Function Call Validation? (and update file name)

Reply:
I've got some where I check the body against the signature too.